cluster-autoscaler/core/scale_down.go

```diff
@@ -832,9 +833,12 @@ func cleanUpNodeAutoprovisionedGroups(cloudProvider cloudprovider.CloudProvider)
 			return err
 		}
 		if size == 0 {
+			ngId := nodeGroup.Id()
 			if err := nodeGroup.Delete(); err != nil {
 				return err
 			}
+			logRecorder.Eventf(apiv1.EventTypeNormal, "DeletedNodeGroup",
```

Comment:
nit: I'd go with "NodeGroupDeleted" (see below comment). Feel free to ignore it though